.goreleaser.yaml

```diff
@@ -38,6 +38,21 @@ builds:
     goarm:
       - 6
       - 7
+    overrides: # Disable CGO on arm platforms, means that we don't support emulated TPM under arm
+      - goarch: arm
+        goos: linux
+        goarm: 6
+        env:
+          - CGO_ENABLED=0
+      - goarch: arm
+        goos: linux
+        goarm: 7
```

Comment:
armv7 ?!

not sure why these override are here, but we don't support arm-32bit

Reply:
yeah, we should probably drop the arm 32-bit already. It was in here because it was mainly free to build the binaries for it, but now that its starting to give issues, it better be dropped.